### DISCUSSION TITLE
How to test react grid layout with react testing library

### DISCUSSION THREAD

**Discussion by TathagatIVP** [react-grid-layout/react-grid-layout#1934] [General]
I have rendered the grid on the screen but react grid layout element has no children. I have used jest-preview to check the ui and nothing is coming up on the screen. 
Does anyone know how can we test/ writes test cases using react testing library

**Comment by JMGonzalezR** [comment]
@TathagatIVP  did you find a way to do this ?

**Comment by TathagatIVP** [comment]
@JMGonzalezR  Hey, I couldn't find any way.

**Comment by suiramdev** [comment]
I haven't found a solution either. This should be considered. UP!
